.golangci.yml

```diff
@@ -4,6 +4,9 @@ linters-settings:
   gosec:
     excludes:
       - G601
+      # G115 is reporting false positives in golangci-lint v1.60.3.
```

Comment:
G115 is reporting false negatives, and the gosec community still hasn't reached a final conclusion. Therefore, I’ve decided to disable it by default: https://github.com/securego/gosec/issues/1212.